app/views/hub/efile_submissions/_log.html.erb

```diff
@@ -11,8 +11,19 @@
           <%= error.message %>
         </div>
     <% end %>
-    <% if transition.metadata["indicators"].present? %>
-      <div> Fraud indicators: <%= transition.metadata["indicators"].map(&:humanize).join(", ") %></div>
+    <% if transition.to_state.in? ["bundling", "fraud_hold"] %>
+      <% fraud_score = transition.efile_submission&.fraud_score %>
+      <div> Fraud Score: <%= fraud_score.score %></div>
+      <% if fraud_score.snapshot.present? %>
+        <div class="accordion spacing-above-10">
+          <a href="#" class="accordion__button" aria-expanded="true" aria-controls="a2">
+            <h3>Details</h3>
+          </a>
+          <div class="accordion__content" id="a2">
+            <%= render partial: "shared/fraud_score_snapshot", locals: { fraud_score: transition.efile_submission.fraud_score, fraud_indicators: Fraud::Indicator.all } %>
```

Comment:
i dont think that it will run multiple queries, because the query gets called only once because it's cached to run the query only once per page. But yes, it probably makes more sense just to load them once.

Since we're going to be grabbing every entry in this table often, I was thinking about using redis cache to store this but we don't have a cache store set up on production so im going to defer that work for now.